src/main/java/com/titanic/airbnbclone/repository/AccommodationRepository.java

```diff
@@ -57,15 +55,17 @@
         return resultList;
     }
 
-    public List<AccommodationResponseDto> getInitAccommodation() {
+    public List<InitAccommodationResponseDto> getInitAccommodation() {
         String queryString = "select distinct a from Accommodation as a left join fetch a.pictures";
         List<Accommodation> accommodations = entityManager
                 .createQuery(queryString, Accommodation.class)
                 .getResultList();
-        accommodations = accommodations.subList(0, 30);
+        int startIndex = 0;
+        int accommodationCount = 30;
```

Comment:
숫자로 변수 줄때 따로 메서드를 두거나 환경변수로 설정하면 더욱 유지보수하기 쉬워질 듯 합니다.

Reply:
메서드로 두거나 환경변수로 설정한다는 게 어떻게 하는걸까요??

Reply:
필드 상단에 final로 적거나 아니면 yml, properties 파일에 적는거죠!
```suggestion
        reservation :
          startIndex: 0
          count: 30
```
이후에 `@ConfigurationProperties`를 사용하거나, 또는 `@Value`를 이용하여 환경변수 값을 가져옵니다
[ConfigurationProperties 사용법](https://www.baeldung.com/configuration-properties-in-spring-boot)

Reply:
이렇게 하면 유지보수하기 편합니다!